android/build.gradle

```diff
@@ -1,27 +1,38 @@
+import org.apache.tools.ant.taskdefs.condition.Os
+
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
     ext {
-        buildToolsVersion = "29.0.3"
+        buildToolsVersion = "31.0.0"
         minSdkVersion = 21
-        compileSdkVersion = 30
-        targetSdkVersion = 30
+        compileSdkVersion = 31
+        targetSdkVersion = 31
```

Comment:
We just realized that our Android SDK doesn't work well on Android 12, so let's keep `targetSdkVersion` to `30` till we fix all issues for Android 12

Reply:
fixed. Changed `targetSdkVersion` to 30